src/test/checkSession.test.ts

```diff
@@ -2,38 +2,30 @@ import { user } from '../mocks/user'
 import { sessionStorage } from '../mocks/sessionStorage'
 import { supabaseStrategy } from '../mocks/authenticator'
 import { authenticatedReq } from '../mocks/requests'
-import { SESSION_ERROR_KEY, SESSION_KEY } from '../mocks/constants'
+import { SESSION_KEY } from '../mocks/constants'
 import { validResponse } from '../mocks/handlers'
 
 describe('[external export] revalidate', async() => {
   it('should redirect if cookie is not set', async() => {
     expect.assertions(1)
-    await supabaseStrategy.checkSession(new Request(''), sessionStorage,
+    await supabaseStrategy.checkSession(new Request(''),
       {
-        sessionKey: SESSION_KEY,
         failureRedirect: '/login',
-        sessionErrorKey: SESSION_ERROR_KEY,
       },
     ).catch(res => expect(res.status).toBe(302))
```

Comment:
maybe add the following expect as well `expect(error.headers.get('Location')).toEqual('/login')`